docs/cookbook/prompt_engineering/chaining_based/self_verification.md

```diff
@@ -89,19 +90,20 @@ async def fill_in_value(solution: str, masked_prompt: str): ...
 class PromptComparison(BaseModel):
     score: int = Field(
         ...,
-        description="""The number of variation prompts which are
-                    semantically equivalent to the original prompt.
-                    For numbers in the prompts, expect the same values, and for
-                    quantitative words, only count semantically identical terms, such
+        description="""The number of variation prompts which are \
+                    semantically equivalent to the original prompt. \
```

Comment:
noticing same thing here and every where else you're doing this, the tab will be included now. Better to just keep this on a single line so you don't have to do a bunch of cleanup (and I think it's fine since the user can scroll), but up to you to figure this out